nf_core/pipeline-template/{{cookiecutter.name_noslash}}/main.nf

```diff
@@ -95,23 +89,23 @@ if( workflow.profile == 'awsbatch') {
 }
 
 // Stage config files
-ch_multiqc_config = Channel.fromPath(params.multiqc_config)
-ch_output_docs = Channel.fromPath("$baseDir/docs/output.md")
+ch_multiqc_config = file(params.multiqc_config, checkIfExists: true)
+ch_output_docs = file("$baseDir/docs/output.md", checkIfExists: true)
 
 /*
  * Create a channel for input read files
  */
-if(params.readPaths){
-    if(params.singleEnd){
+if (params.readPaths) {
+    if (params.singleEnd) {
         Channel
             .from(params.readPaths)
-            .map { row -> [ row[0], [file(row[1][0])]] }
+            .map { row -> [ row[0], [ file(row[1][0], checkIfExists: true) ] ] }
```

Comment:
As above, don't think that this will work on `file`..?

```suggestion
            .map { row -> [ row[0], [ file(row[1][0]) ] ] }
```

Reply:
Ok nice! So `checkIfExists` does work for `file` then.. Sorry, from how I read the comments around this feature, I thought it was _only_ for `Channel.fromPath`. In that case by all means keep it in place... 👍